src/api/user.js

```diff
@@ -24,6 +24,27 @@ const getUser = async ({ userId, token }) => {
   return results;
 };
 
+/**
+  * Get a user's profile based on user's db id
+  * @param {string} token - User's auth0 token
+  * @param {string} userId - user's db id
+  * @returns {Promise} - resolves to user object
+*/
+const getUserProfile = async (userId, token) => {
+  const url = new URL(process.env.REACT_APP_API_HOST);
+  url.pathname = `/api/v1/users/${userId}?include=identites`;
```

Comment:
Did the include=identities not work?

Reply:
I'm able to see them included as an empty array when I test it in postman, so I'm thinking it is working but maybe I'm not submitting them correctly when creating a user. For reference I'm submitting an object that looks like this. All the other fields write to the backend but the identities.
<img width="308" alt="Screen Shot 2021-04-06 at 8 58 50 PM" src="https://user-images.githubusercontent.com/17222004/113795539-2aa06d00-971b-11eb-8acc-9b5556d43b77.png">